Release/src/CMakeLists.txt

```diff
@@ -128,9 +128,10 @@ endif()
 if(CPPREST_HTTP_CLIENT_IMPL STREQUAL "asio")
   cpprest_find_boost()
   cpprest_find_openssl()
-  target_compile_definitions(cpprest PUBLIC -DCPPREST_FORCE_HTTP_CLIENT_ASIO)
+  cpprest_find_botan()
+  target_compile_definitions(cpprest PUBLIC -DCPPREST_BOTAN_SSL -DCPPREST_FORCE_HTTP_CLIENT_ASIO)
```

Comment:
Unconditionally switching customers to Botan from OpenSSL doesn't seem like a reasonable change. If you want this to be maintained I think you need to (1) add it as an option rather than unconditionally enable it, and (2) add a configuration verifying that it works to azure-pipelines.yaml. Otherwise maintainers who have never used Botan and have no idea how it works will break it on a regular basis.